autogpt/llm/chat.py

```diff
@@ -27,7 +28,7 @@ def create_chat_message(role, content) -> Message:
     Returns:
     dict: A dictionary containing the role and content of the message.
     """
-    return {"role": role, "content": content}
+    return {"role": role, "content": content, "id": uuid4().hex}
```

Comment:
We should try just an autoincrementing integer - would be simpler and would make the sequence more obvious